spec/controllers/media_objects_controller_spec.rb

```diff
@@ -812,31 +812,31 @@
               FactoryBot.create(:checkout, media_object_id: media_object.id, user_id: controller.current_user.id)
               get :show, params: { id: media_object.id }
               expect(response).to render_template(:_share_resource)
-              expect(response).to render_template(:_embed_resource)
+              expect(response).not_to render_template(:_embed_resource)
               expect(response).to render_template(:_lti_url)
             end
             it "managers: should include lti, embed, and share" do
```

Comment:
Can you please update the labels for these tests so they reflect the tests now?

Reply:
Oh, yep. Sorry about that.